doc/on_chain_spec.md

```diff
@@ -79,39 +80,47 @@ is statically parametrized by the auction terms.
 Similarly, all the validator scripts in the auction are made unique
 via static parametrization on the auction terms.
 
-Between the auction announcement and the bidding start time,
-the seller fixes the list of bidders
```

Comment:
Line about approval disappeared completely. While this paragraph probably should explain that still or it is not clear why seller will provide any signed data at all.